cognite/neat/app/api/routers/rules.py

```diff
@@ -105,9 +105,9 @@ def get_rules(
 
 @router.get("/api/rules/from_file")
 def get_original_rules_from_file(file_name: str):
-    """Endpoing for retrieving raw transformation from file"""
+    # """Endpoint for retrieving raw transformation from file"""
     path = Path(NEAT_APP.config.rules_store_path) / file_name
-    rules = parse_rules_from_excel_file(path)
+    rules = cast(Rules, importer.ExcelImporter(filepath=path).to_rules(return_report=False, skip_validation=False))
```

Comment:
Surprising that this requires casting. Shouldn't the to_rules return a Rules instance? Missing overload?